clickhouse/base/wire_format.cpp

```diff
@@ -99,4 +100,82 @@ bool WireFormat::SkipString(InputStream& input) {
     return false;
 }
 
+const std::vector<char> quoted_chars = {'\0', '\b', '\t', '\n', '\'', '\\'};
```

Comment:
please use `const char quoted_chars[] = ...` instead of `std::vector`, and move this into `find_quoted_chars`, no need to have a global symbol. That would also help performance :)